src/renderer/viz/expressions/buckets.js

```diff
@@ -71,6 +71,8 @@ export default class Buckets extends BaseExpression {
             looseType = input.type;
         }
 
+        checkType('buckets', 'list', 1, ['number-array', 'category-array'], list);
```

Comment:
The check should be on `_bindMetadata` because in the constructor you don't know the type of references to variables

Reply:
Thx David!, I saw a similar one on that method, but I wasn't sure

Reply:
Done! I have also fixed the constructor accordingly